src/LightInject/LightInject.cs

```diff
@@ -4829,7 +4829,7 @@ private T[] CreateSnapshot()
 
         private class PropertyDependencyDisabler : IPropertyDependencySelector
         {
-            public IEnumerable<PropertyDependency> Execute(Type type)
+            public IEnumerable<PropertyDependency> Execute(Registration registration)
```

Comment:
This is also a public interface and it would be a breaking change to change the signature.
For instance, it would break here

https://github.com/seesharper/LightInject.Annotation/blob/bef7c548d430f5749ccbf8486a798d364e534502/src/LightInject.Annotation/LightInject.Annotation.cs#L110